[quant][fx] Lower operator.matmul in convert_fx

andrewor14 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #113954

Summary: We support lowering `torch.matmul` but not
`operator.matmul`. This commit adds support for the latter,
which enables lowering the shorthand `@`. This address
https://github.com/pytorch/pytorch/issues/111450.

Test Plan:
python test/test_quantization.py TestQuantizeFx

Reviewers: jerryzh168

Subscribers: jerryzh168, supriyar

andrewor14 committed:
[quant][fx] Lower operator.matmul in convert_fx

Summary: We support lowering `torch.matmul` but not
`operator.matmul`. This commit adds support for the latter,
which enables lowering the shorthand `@`. This address
https://github.com/pytorch/pytorch/issues/111450.

Test Plan:
python test/test_quantization.py TestQuantizeFx

Reviewers: jerryzh168

Subscribers: jerryzh168, supriyar

[ghstack-poisoned]

andrewor14 committed:
Update on "[quant][fx] Lower operator.matmul in convert_fx"


Summary: We support lowering `torch.matmul` but not
`operator.matmul`. This commit adds support for the latter,
which enables lowering the shorthand `@`. This address
https://github.com/pytorch/pytorch/issues/111450.

Test Plan:
python test/test_quantization.py TestQuantizeFx

Reviewers: jerryzh168

Subscribers: jerryzh168, supriyar

[ghstack-poisoned]

jerryzh168 reviewed (approved):
do we need to update test as well?

andrewor14 commented:
I didn't find tests specifically for lowering the other binary ops either. I think I'll just add a TODO here

andrewor14 committed:
Update on "[quant][fx] Lower operator.matmul in convert_fx"


Summary: We support lowering `torch.matmul` but not
`operator.matmul`. This commit adds support for the latter,
which enables lowering the shorthand `@`. This address
https://github.com/pytorch/pytorch/issues/111450.

Test Plan:
python test/test_quantization.py TestQuantizeFx

Reviewers: jerryzh168

Subscribers: jerryzh168, supriyar

[ghstack-poisoned]

andrewor14 commented:
@pytorchbot merge
